www/js/controllers/splash_controller.js

```diff
@@ -1,5 +1,15 @@
-angular.module('mobility')
+(function() {
+  'use strict';
 
-.controller('SplashController', function($scope) {
-  $scope.splashText = 'Splash Page';
-});
+  angular
+    .module('mobility')
+    .controller('SplashController', SplashController);
+
+  SplashController.$inject = ['$scope'];
+
+  function SplashController($scope) {
```

Comment:
Left it in there in case we needed it for later.  Nevertheless, I removed it for now.